con-base/src/main/java/ba/codecentric/base/domain/Hotel.java

```diff
@@ -62,6 +57,14 @@
     @Size(max = 500, message = "Description is too long, 500 characters allowed")
     private String description;
 
+    @NotNull(message = "Please add Check-in time")
+    @Column(name = "check_in")
+    private Time checkIn;
```

Comment:
How about to try with Java 8 `LocalTime`? The simple reason is that the time can be easly formatted. That way, you can get rid of the seconds For each field, you'd probably want to create a getter which returns a formatted time as a string. And setter/constructor which accepts string and parses it to `LocalTime`. 
P.S. Take a look at [DateTimeFormatter docs](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html).